Test/Utils/ComputeNode/Service.ps1

```diff
@@ -97,3 +104,81 @@ function Out-StdoutAndStderrToLogFile {
 
     Write-Log $Output.Output
 }
+
+function Get-AgentServiceName {
+    return "contrail-vrouter-agent"
+}
+
+function Get-AgentExecutablePath {
+    return "C:\Program Files\Juniper Networks\agent\contrail-vrouter-agent.exe"
+}
+
+function Get-AgentServiceStatus {
+    Param (
+        [Parameter(Mandatory=$true)] $Session
+    )
+
+    Get-ServiceStatus -Session $Session -ServiceName (Get-AgentServiceName)
+}
+
+function New-AgentService {
+    Param (
+        [Parameter(Mandatory=$true)] $Session
+    )
+
+    $LogDir = Get-ComputeLogsDir
+    $LogPath = Join-Path $LogDir "contrail-vrouter-agent-service.log"
+    $ServiceName = Get-AgentServiceName
+    $ExecutablePath = Get-AgentExecutablePath
+
+    $ConfigPath = Get-DefaultAgentConfigPath
+    $ConfigFileParam = "--config_file=$ConfigPath"
+
+    Install-ServiceWithNSSM -Session $Session `
+        -ServiceName $ServiceName `
+        -ExecutablePath $ExecutablePath `
+        -CommandLineParams @($ConfigFileParam)
+
+    Out-StdoutAndStderrToLogFile -Session $Session `
+        -ServiceName $ServiceName `
+        -LogPath $LogPath
+}
+
+function Start-AgentService {
+    Param (
+        [Parameter(Mandatory=$true)] $Session
+    )
+
+    Write-Log "Starting Agent"
+
+    $AgentServiceName = Get-AgentServiceName
+    $Output = Invoke-NativeCommand -Session $Session -ScriptBlock {
+        $Output = netstat -abq  #dial tcp bug debug output
+        #TODO: After the bugfix, use Enable-Service generic function here.
```

Comment:
Which bugfix?

Reply:
bugfix for `dial tcp 127.0.0.1:9091: connectex: No connection could be made because the target machine actively refused it.` bug

Reply:
OK so please write it in comment so that we know in the future